irohad/torii/impl/command_service_transport_grpc.cpp

```diff
@@ -150,7 +163,20 @@ namespace iroha {
             });
       }
 
-      return grpc::Status::OK;
+      auto status = grpc::Status::OK;
+      if (not unprocessed_batches.empty()) {
+        std::stringstream message;
+        message << "The batches with the following reduced hashes were not "
+                   "processed, please resend them: [";
```

Comment:
parsing such a string on clients to get the unprocessed batches is not quite a good idea. I suggest extending the API to add a `ListTorii` call with status response.